### PR TITLE
Add a flag to be able to skip updating "last known good"

### PR DESCRIPTION
**Pull request by samturci**

### CODE REVIEW

File: cmd/reporter/reporter.go [hunk 1]
```diff
@@ -29,14 +29,15 @@ import (
...
-	verbose    = flag.Bool("verbose", false, "print additional output for each file which changed")
+	configFile     = flag.String("config-file", "", "required report config file to use")
+	walkPath       = flag.String("walk-path", "", "path to search for Walks")
+	reviewFile     = flag.String("review-file", "", "path to the file containing a list of last-known-good states - this needs to be writeable")
+	hostname       = flag.String("hostname", "", "host to review the differences for")
+	beforeFile     = flag.String("before-file", "", "path to the file to compare against (last known good typically)")
+	afterFile      = flag.String("after-file", "", "path to the file to compare with the before state")
+	paginate       = flag.Bool("paginate", false, "pipe output into $PAGER in order to paginate and make reviews easier")
+	verbose        = flag.Bool("verbose", false, "print additional output for each file which changed")
+	noReviewUpdate = flag.Bool("no-review-update", false, "don't ask to update the \"last known good\" review")
```
Comment (x1ddos) [inline cmd/reporter/reporter.go]: Sometimes double negation is harder to understand. It's clear in the flag description here but in the code below `if !*noReviewUpdate` might be a bit confusing. How about `skip-review-update` ? It's only 2 characters longer.

Or the opposite: `update-review` flag wich defaults to `true`. Then one can `-update-review=false`.
Alternatively, make it a breaking change and have `update-review` default to `false`. Then one needs to explicitely say `-update-review=true`. I guess it depends on which mode is used more frequently. Our team would use `-update-review=false` most of the time. OTOH we run it from a script which always sets appropriate flags.

We've just made a breaking change with flags name anyway if you're worried about backwards compatibility.

I'm also fine with leaving this as is if you think it's best.

Comment (samturci) [reply cmd/reporter/reporter.go]: I like doing the opposite and having it false by default if that's fine.

File: cmd/reporter/reporter.go [hunk 2]
```diff
@@ -153,7 +153,7 @@ func main() {
 	}
 
 	// Update reviews file if desired.
-	if !*noReviewUpdate && updateReviews() {
+	if *updateReview && askUpdateReviews() {
```
Comment (x1ddos) [inline cmd/reporter/reporter.go]: I like this much better!

**Comment by x1ddos** [comment]
Also, I think I like this multi-commit review better. It lets me add comments.
But maybe you prefer to squash before merging. Up to you.

**Comment by x1ddos** [comment]
Also forgot here. We'll need to tag this merge commit with "v0.2.0" to indicate the breaking change.

**Comment by x1ddos** [comment]
Pushed some tags:

```sh
$ git show-ref --tag
caf7eb541875506c28a28499b5a3c6d11a6c6a90 refs/tags/v0.0.1
6dbc4ebe2b7e0ee75efb3af3468b8acd82bbb397 refs/tags/v0.1.0
2a5c8f28a14f4a696d46bb02db5abb3b73125366 refs/tags/v0.2.0
```

**Pull request merged**